### DISCUSSION TITLE
Add quick link to views (homepage) when in nested config path

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#12470] [Dashboard] [answered]
As per title. Always a real pain to have to click your way back to the Frontend views after having been in de deeper layers of the config menu

please add a link/icon in the top bar? So we can immediately go back to the views

especially so on mobile
Thanks for considering

**Comment by zsarnett** [comment, marked as answer]
What do you mean by Frontend Views? Are you saying Dashboards?

Why not use the sidebar?

**Reply by Mariusthvdb** [reply]
I stand corrected and am completely ashamed now....
Never realized we could do that on mobile.
Tsss
